docs/architecture/psa-crypto-implementation-structure.md

```diff
@@ -49,7 +49,7 @@ psa_status_t psa_api( ... )
 ```
 The code of most PSA APIs is expected to match precisely the above layout. However, it is likely that the code structure of some APIs will be more complicated with several calls to the driver interface, mainly to encompass a larger variety of hardware designs. For example, to encompass hardware accelerators that are capable of verifying a MAC and those that are only capable of computing a MAC, the psa_mac_verify() API could call first psa_driver_wrapper_mac_verify() and then fallback to psa_driver_wrapper_mac_compute().
 
-The implementations of `psa_driver_wrapper_<entry_point>` functions are generated by the build system based on the JSON driver description files of the various PSA drivers making up the Mbed TLS PSA Cryptography API implementation. The implementations are generated in a psa_crypto_driver_wrappers.c C file and the function prototypes declared in a psa_crypto_driver_wrappers.h header file.
+The implementations of `psa_driver_wrapper_<entry_point>` functions are generated by the build system based on the JSON driver description files of the various PSA drivers making up the Mbed TLS PSA Cryptography API implementation. The implementations are generated in a psa_crypto_driver_wrappers.c C file and the function prototypes declared in a psa_crypto_driver_wrappers_no_static.h header file.
```

Comment:
I don't think it's correct that "The implementations are generated in a psa_crypto_driver_wrappers.c C file", is it?

There's also "Driver dispatch code in `scripts/data_files/driver_templates/psa_crypto_driver_wrappers.c.jinja`" at line 142

Reply:
Ah, that might be old build artefacts left behind in the working directory